packages/ui/common/src/lib/environments/environment.ts

```diff
@@ -1,7 +1,9 @@
 export const environment = {
   production: false,
-  apiUrl: 'http://localhost:3000/v1',
+  apiUrl:
```

Comment:
@obenazouz-wedof can you please revert these changes?

Reply:
done :) ! thank you that was a little mistake